docker-compose.yml

```diff
@@ -20,6 +20,8 @@ notarysigner:
    - notarymysql
   command: -config=fixtures/signer-config.json
 notarymysql:
+  volumes:
+    - notarymysql:/var/lib/mysql
```

Comment:
Actually, this may already be the case; https://github.com/docker/compose/blob/master/docs/compose-file.md#volumes-volume_driver. Is the intent here to create a "named" volume, or a bind-mount?